.circleci/config.yml

```diff
@@ -13,6 +21,54 @@ aliases:
 
 version: 2
 jobs:
+  drupal_update_dependencies: &update_dependencies
+    <<: *drupal_defaults
+    steps:
+      - checkout
+      # Restore Composer Cache
+      - restore_cache:
+          keys:
+            - v4-dependencies-{{ checksum "custom/admin_ui_support/composer.json" }}
+            - v4-dependencies-
+      - run:
+          working_directory: /var/www/html
+          command: |
+            ./update-dependencies.sh $CIRCLE_PROJECT_REPONAME
+      # Add JSON API
+      - run:
+            working_directory: /var/www/html
+            command: /usr/local/bin/composer require drupal/jsonapi:^1.16
```

Comment:
I think this can be removed, as you have (properly) added jsonapi as a requirement in composer.json below.

Reply:
That seems to make sense

Reply:
Just a followup on this. I tried to remove this and JSON API was not actually installed.